### PR TITLE
Add part and quantity tables to each major step in the docs

### PR DESCRIPTION
**Pull request by jazzychad**

### CODE REVIEW

**Comment by jazzychad** [comment]
I double checked each part and quantity, but probably good to have another person/people audit the numbers to make sure they are correct.

**Comment by jazzychad** [comment]
Also, feel free to suggest different format, etc.. this is just something I think would be extremely useful to have while assembling since reading through the instructions, it can be hard to pick out exactly what/how many of each part are needed. Nice to know a priori what you need to have in front of you.

File: doc/content/en/docs/mechanical-assembly/mount-x-to-y/index.md [hunk 1]
```diff
@@ -39,6 +47,15 @@ Now that we have both gantries assembled, it's time to put them together.
 
 ## Camera and Ring Light
 
+| Qty | Part                        |
+|----:|-----------------------------|
+|   1 | USB Camera                  |
```
Comment (atanisoft) [inline doc/content/en/docs/mechanical-assembly/mount-x-to-y/index.md:52]: probably best to indicate which camera (top?)

File: doc/content/en/docs/mechanical-assembly/populating-staging-plate/index.md [hunk 2]
```diff
@@ -17,6 +17,12 @@ Refer to the images below for referencing where to mount parts on the staging pl
 
 ## Soldering Pump
 
+| Qty | Part             |
+|----:|------------------|
+|   1 | Vacuum Pump      |
+|   1 | Vacuum Pump Wire |
+|   2 | Heatshrink       |
```
Comment (atanisoft) [inline doc/content/en/docs/mechanical-assembly/populating-staging-plate/index.md:24]: indicate size as there are two different sizes included in the kit.

File: doc/content/en/docs/mechanical-assembly/mount-x-to-y/index.md (hunk 1 above)
Comment (jazzychad) [reply doc/content/en/docs/mechanical-assembly/mount-x-to-y/index.md:52]: good point

File: doc/content/en/docs/mechanical-assembly/populating-staging-plate/index.md (hunk 2 above)
Comment (jazzychad) [reply doc/content/en/docs/mechanical-assembly/populating-staging-plate/index.md:24]: will do

**Comment by jazzychad** [comment]
Made changes addressing @atanisoft comments

**Comment by sphawes** [comment]
@jazzychad thank you so much, this is incredible. Going to merge this in for the next docs update!

**Pull request merged**